packages/node_modules/@webex/test-users/src/index.js

```diff
@@ -0,0 +1,290 @@
+import assert from 'assert';
+
+import btoa from 'btoa';
+import _ from 'lodash';
+import randomName from 'node-random-name';
+import {request} from '@webex/http-core';
+
+import generateRandomString from './generate-random-string.js';
+
+
+const BASE_PATH_SECURE = '/users/test_users_s';
+const BASE_PATH = '/users/test_users';
+
+/**
+ * Computes `expires` and `refresh_token_expires` from `expires_in` and
+ * `refresh_token_expires_in` and creates an `authorization` string.
+ * @param {Object} token
+ * @private
+ * @returns {Object}
+ */
+function fixToken(token) {
+  const now = Date.now();
+
+  if (token.expires_in && !token.expires) {
+    token.expires = now + token.expires_in * 1000;
+  }
+
+  if (token.refresh_token_expires_in && !token.refresh_token_expires) {
+  /* eslint camelcase: [0] */
+    token.refresh_token_expires = now + token.refresh_token_expires_in * 1000;
+  }
+
+  if (token.token_type && token.access_token) {
+    token.authorization = `${token.token_type} ${token.access_token}`;
+  }
+
+  return token;
+}
+
+let clientToken;
+
+/**
+ * Fetches credentials to talk to the test_users_s endpoint
+ *
+ * Caches result in `clientToken` variable for multiple runs
+ * @param {Object} options
+ * @param {string} options.clientId
+ * @param {string} options.clientSecret
+ * @param {string} options.idbrokerUrl
+ * @private
+ * @returns {String}
+ */
```

Comment:
Should we change this to a `@typedef` for `options`?

Reply:
I didn't create one for this because it really is just a subset of the required params from `CreateUserOptions` typedef.